Client/sdk/game/RenderWare.h

```diff
@@ -321,25 +321,35 @@ struct RwGeometry
     unsigned char  unknown1[14];
     unsigned short refs;
 };
-struct RpInterpolation
+
+/* Interpolator flags */
+enum RpInterpolatorFlag : int32
+{
+    rpINTERPOLATORDIRTYINSTANCE = 0x01,
+    rpINTERPOLATORDIRTYSPHERE = 0x02,
+    rpINTERPOLATORNOFRAMEDIRTY = 0x04,
+};
+struct RpInterpolator
 {
-    unsigned int unknown1;
-    unsigned int unknown2;
-    float        unknown3;
-    float        unknown4;
-    float        unknown5;
+    int32 flags; /**< flags */
+    int16 startMorphTarget; /**< startMorphTarget */
+    int16 endMorphTarget; /**< endMorphTarget */
+    float time; /**< time */
+    float recipTime; /**< recipTime */
+    float position; /**< position */
```

Comment:
Please use `//<` for the documentation strings of member fields.
Also: use fixed size integer types with the `_t` suffix.